### ISSUE TITLE
Images in ckEditor and ckFinder

### ISSUE THREAD

**Issue by aaltenbu** [blueriver/MuraCMS#2010]
Images uploaded from the ckeditor going to the folder /assets/Image/. Is there a reason not using the same folder as images uploaded as associated images: /cache/file/?
Image properties e.g. image sizes are not the same as defined in Site Config.
Would be nice if inline images having the same behaviour and processing.
Thanks

**Comment by mattlevine** [comment]
Yes, I basically think of it as Mura having two classes of assets loose and structured.  

When you uploaded a file as a file content node or as an associated image to a content node you are never supposed to interacted with that directly.  The name of the file is the same as it fileid in the database and it is a apart of the structured data that defined that content node or even just a particular version with a content node version history. Those are what you find in the cache directory and are what I call structured files.   You would just cause problems if you tried to manually manage them.

Loose files are the ones uploaded ckfinder and added as link in ckeditor.   Mura has no actually knowledge about them. They are just used in the the html that makes up html that is stored as a string in the database.  In the assets directory you can feel free to create directories and place whatever you want in it.

I'm not totally happy how it is, but at this point I'm not sure what to do simplify it.  If anything one day I could see making all files structured and then making is so that when you click to upload files in CKEditor that is would actually store the uploaded asset as a Mura file content node.  However, I think people will always want the freedom to just upload a file via ckfinder or ftp and then have a tool from within CKEditor to select it.  

Thanks,
-Matt
